docs/deployment/FPC_on_Azure.md

```diff
@@ -0,0 +1,139 @@
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+sudo apt-get update
+sudo apt -y install libssl-dev libsgx-enclave-common libsgx-enclave-common-dev libsgx-ae-qe3 libsgx-ae-qve libsgx-epid libsgx-launch libsgx-pce-logic libsgx-qe3-logic libsgx-quote-ex libsgx-uae-service libsgx-urts
+sudo reboot
+```
+After reboot is completed connect to your vm again.
+```bash
+#check if asemd service is running
+sudo systemctl status aesmd.service
+```
+```bash
+sudo usermod -aG docker $(whoami)
+go env -w GO111MODULE=auto
+
+echo 'export GOPATH=$HOME/go' >> .bashrc
+echo 'export GOBIN=$GOPATH/bin' >> .bashrc
+echo 'export PATH=$PATH:$GOBIN' >> .bashrc
+
+sudo reboot
+``` 
+After reboot is completed connect to your vm again.
+```bash
+mkdir -p $GOPATH
+cd $GOPATH
+go get github.com/hyperledger/fabric-private-chaincode
+cd  ~
+echo 'export FPC_PATH=$GOPATH/src/github.com/hyperledger/fabric-private-chaincode' >> .bashrc
+#you can also re-log than you don't need to set it
+export FPC_PATH=$GOPATH/src/github.com/hyperledger/fabric-private-chaincode 
+```
+
+## Registering for the SGX Attestation Service Utilizing EPID
+* Get a account [here](https://www.intel.com/content/www/us/en/forms/basic-intel-registration.html)
+* Sign in.
+* Once you are signed in go [here](here: https://api.portal.trustedservices.intel.com/EPID-attestation)
+* Subscribe for development unlikable.
+* You will receive a SPID, Primary Key and Secondary Key.
+* Use these information to replace the appropriate places in the commands below.
+
+```bash
+echo '[YOUR_SPID]' > ${FPC_PATH}/config/ias/spid.txt
+echo '[YOUR_PRIMARY_KEY]' > ${FPC_PATH}/config/ias/api_key.txt
+echo 'epid-unlinkable' > ${FPC_PATH}/config/ias/spid_type.txt
+```
+
+### Known Issues
+If you don't have an account and directly try to *Subscribe* to the EPID-attestation
+[here](https://api.portal.trustedservices.intel.com/EPID-attestation). ![Subscribe to
+EPID-attestation api](pictures/SubscribeButton.png) And you are not signed in already with the account
+trough [this](https://www.intel.com/content/www/us/en/forms/basic-intel-registration.html) then you
+will be lead to the page. ![broken account creation link](pictures/CreateIDZAccount.png)
+If you then click on *create new IDZ account* you will land on a
+[page](https://software.intel.com/registration/?lang=en-us) saying that the site is held hostage.
+Hence, you won't be able to register an account here. So please us the site
+[here](https://www.intel.com/content/www/us/en/forms/basic-intel-registration.html) to register
+yourselfe and just sign in on the top right on the [api
+page](https://api.portal.trustedservices.intel.com/EPID-attestation) with the previously created
+credentials.
+
+## Building the FPC dev container
```

Comment:
As mentioned already above. This section refers to setup the FPC development environment. Maybe mention the two options already described in our main readme, and say, in this guide we continue with the docker-based option. :)

Reply:
sure